File: card-api/src/main/java/com/jean202/cardmizer/api/CardMizerApplication.java
package com.jean202.cardmizer.api;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = "com.jean202.cardmizer")
public class CardMizerApplication {
    public static void main(String[] args) {
        SpringApplication.run(CardMizerApplication.class, args);
    }
}

File: card-api/src/main/kotlin/com/jean202/cardmizer/api/CardMizerApplication.kt
package com.jean202.cardmizer.api

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.jean202.cardmizer"])
class CardMizerApplication

fun main(args: Array<String>) {
    runApplication<CardMizerApplication>(*args)
}
